backend/src/Squidex.Domain.Apps.Entities/Contents/DomainObject/Guards/GuardContent.cs

```diff
@@ -174,5 +181,19 @@ private static async Task ValidateCanUpdate(IContentEntity content, IContentWork
                 throw new DomainException(T.Get("contents.workflowErrorUpdate", new { status }));
             }
         }
+
+        public static void CheckPermission(IContentEntity content, ContentCommand command, string permission)
+        {
+            if (content.CreatedBy != null && content.CreatedBy.Equals(command.Actor))
+            {
+                return;
+            }
+
+            var requiredPermission = Permissions.ForApp(permission, content.AppId.Name, content.SchemaId.Name);
+            if (!command.User.Permissions().Allows(requiredPermission))
+            {
+                throw new DomainForbiddenException("...");
```

Comment:
You should provide a message here, I said it is not required but I was wrong. Check the translation file if we have something that makes sense.